config/database.yml

```diff
@@ -58,7 +58,7 @@ development:
 test:
   <<: *default
   database: WarAndPiece_test
-
+  #template: template0
```

Comment:
Let's remove these lines from this file.

Reply:
I pulled it out but do I just keep adding it? i just get error without it